src/components/Footer/Footer.scss

```
@@ -0,0 +1,403 @@
+    background: #f4f4f4;
+
+
+    .kakaoWrapper {
+
+      .kakaoTitle {
+        display: flex;
+        align-items: center;
+
+        img {
+          width: 33px;
+          height: 31px;
+          overflow: hidden;
+        }
+      }
+
+      h3 {
+        display: flex;
+        margin-left: 2px;
+        font-size: 13px;
+        font-weight: 700;
+      }
+    }
+
+    .kakaoContent {
+      width: 370px;
+      margin: 5px 0;
+      white-space: normal;
+      font-size: 11px;
+    }
+  }
+
+  .kakaoBtn {
+    height: 30px;
+    margin: 5px 0;
+    padding: 5px 17px;
+    border: 1px solid #545454;
+    border-radius: 27px;
+    font-size: 11px;
+    background: #292929;
+    color: #fff;
+
+    &:hover {
+      border: 1px solid #e5e5e5;
+      background: #fff;
+      color: #000;
+    }
+  }
+
+  .totalStore {
+    display: flex;
+    margin: 15px 0;
+    font-size: 9px;
+    font-weight: 700;
+    color: #545454;
+
+  }
+
+  .storeSearchBtn {
+    padding: 7px 160px;
+    border: 1px solid #f4f4f4;
+    border-radius: 27px;
+    background: #fff;
+  }
+
+  .aboutWrapper {
+    h4 {
+      margin-bottom: 14px;
+      font-size: 9px;
+      font-weight: 600;
+      color: #545454;
+    }
+  }
+
+  .collectionWrapper {
+    h4 {
+      margin-bottom: 14px;
+      font-size: 9px;
+      font-weight: 600;
+      color: #545454;
+    }
+
+  }
+
+  .helpWrapper {
+    h4 {
+      margin-bottom: 14px;
+      font-size: 9px;
+      font-weight: 600;
+      color: #545454;
+    }
+  }
+
+  .list {
+    color: #545454;
+
+    li {
+      margin: 6px 0;
+      font-size: 11px;
+      cursor: pointer;
+
+      &:hover {
+        text-decoration: underline;
+      }
+    }
+  }
+
+  .helps {
+    li {
+      margin: 20px 0;
+
+      strong {
+        display: block;
+        margin: 20px 0;
+        font-size: 9px;
+        font-weight: 600;
+      }
+
+      em {
+        display: block;
+        margin: 20px 0;
+      }
+
+      address {
+        width: 283px;
+        font-size: 10px;
+        white-space: normal;
+      }
+    }
+
+    .bordItem {
+      font-size: 8px;
+      font-weight: 600;
+    }
+
+  }
+
+  .addressWrapper {
+    display: flex;
+    padding: 15px 50px;
+    font-size: 9px;
+    background: #f4f4f4;
+    color: #545454;
+
+    .addressContent {
+      padding-right: 250px;
+    }
+
+    .insuranceContent {
+      padding-right: 110px;
+    }
+  }
+
+  .footerBottom {
+    padding: 20px 40px;
+
+
+    .bottoms {
+      display: flex;
+      font-size: 9px;
+
+      :first-of-type {
+        padding-left: 0;
+        border-left: 0;
+      }
+
+      li {
+        height: 15px;
+        margin-left: 7px;
+        padding-left: 10px;
+        border-left: 1px solid #000;
+      }
+
+      .langeContent {
+        margin-left: auto;
+        border-left: 0;
+
+
+        img {
+          width: 20px;
+          height: 20px;
+        }
+
+        .langeBtn {
+          span {
+            position: relative;
```

전반적으로 불필요한 셀렉터 중첩이 보입니다. 이 경우에도 `.langeContent`로 직접 사용해줄 수 있겠네요.
```suggestion
        .langeContent {
            position: relative;
```

---

종택님 리뷰 반영하여 수정했습니다